This patch makes sure that dojo.i18n.normalizeLocale() will in any case return
a locale in lower-case. If dojo.locale is set to, e.g., "de-DE", this caused
the following error in a build js-file:
Error: Bundle not found: validate in dijit.form , locale=de-DE
--- ./dojo-release-1.6.1-src/dojo/i18n.js.org	2011-09-28 16:17:11.000000000 +0200
+++ ./dojo-release-1.6.1-src/dojo/i18n.js	2011-09-28 16:22:28.000000000 +0200
@@ -77,7 +77,7 @@
 	//		If no locale is specified, the dojo.locale is returned.  dojo.locale is defined by
 	//		the user agent's locale unless overridden by djConfig.
 
-	var result = locale ? locale.toLowerCase() : dojo.locale;
+	var result = locale ? locale.toLowerCase() : dojo.locale.toLowerCase();
 	if(result == "root"){
 		result = "ROOT";
 	}
